update-snapshots fails (3.0) due to missing jupyterlab setup

```
  set -eux
  jlpm
  python -m pip install .
  shell: /usr/bin/bash -e {0}
+ jlpm
/home/runner/work/_temp/883e7265-4e7f-4365-b0ea-df9e52ade86c.sh: line 2: jlpm: command not found
Error: Process completed with exit code 127.
```

Example https://github.com/jupyterlab/ui-profiler/actions/runs/3722798618/jobs/6313896572#logs

Ideally we would test actions.

--- a/{{cookiecutter.python_name}}/.github/workflows/update-integration-tests.yml
+++ b/{{cookiecutter.python_name}}/.github/workflows/update-integration-tests.yml
@@ -29,7 +29,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Base Setup
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+
       - name: Install dependencies
+        run: python -m pip install -U "jupyterlab>=4.0.0a29,<5"
+
+      - name: Install extension
         run: |
           set -eux
           jlpm

--- a/{{cookiecutter.python_name}}/ui-tests/package.json
+++ b/{{cookiecutter.python_name}}/ui-tests/package.json
@@ -5,7 +5,8 @@
   "private": true,
   "scripts": {
     "start": "jupyter lab --config jupyter_server_test_config.py",
-    "test": "jlpm playwright test"
+    "test": "jlpm playwright test",
+    "test:update": "jlpm playwright test --update-snapshots"
   },
   "devDependencies": {
     "@jupyterlab/galata": "^5.0.0-alpha.14"

